spec/controllers/admin_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe AdminController, type: :controller do
+  describe '#authenticate_admin' do
```

Comment:
I think I'd still like Brendan's opinion. Testing a private method is counter to Stembolt convention.